src/pages/game/game.js

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from 'react';
-import GameHeader from '../../components/gameHeaderComponent/gameHeaderComponent';
+import Game from '../../components/boardComponent/boardComponent';
 
 class gamePage extends Component {
     render() {
+        console.log('page 2 loaded')
```

Comment:
Remember to get rid of any `console.logs` before pushing up.

Reply:
Thought I had got them all, just removed this one